packages/encryption/src/image-steganography/index.ts

```diff
@@ -18,6 +18,7 @@ export interface EncodeImageOptions extends SteganographyIO {
 export enum SteganographyPreset {
     Preset2021 = '2021',
     Preset2022 = '2022',
+    Preset2023 = '2023',
```

Comment:
```suggestion
    Preset2022v38 = '2022-v38',
```

Reply:
Now the v37 is 2023, and v38 with v2 is 2022.